js/binance.js

```diff
@@ -5291,12 +5291,8 @@ module.exports = class binance extends Exchange {
         for (let i = 0; i < rows.length; i++) {
             const row = rows[i];
             const timestamp = this.safeNumber (row, 'interestAccuredTime');
-            let account = this.safeSymbol (this.safeString (row, 'isolatedSymbol'));
-            if (account === undefined) {
-                account = 'CROSS';
-            }
             interest.push ({
-                'account': account, // isolated symbol, will not be returned for crossed margin
+                'account': symbol === undefined ? 'CROSS' : symbol,
```

Comment:
This edit violated the following CONTRIBUTING guideline:

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#using-ternary-conditionals

Reply:
We have to wrap it in brackets like this:

```JavaScript
'account': (symbol === undefined) ? 'CROSS' : symbol,
```